pkg/bpf/map_linux.go

```diff
@@ -117,7 +117,10 @@ type Map struct {
 	// DumpParser is a function for parsing keys and values from BPF maps
 	DumpParser DumpParser
 
-	cache map[string]*cacheEntry
```

Comment:
I don't think you have to change the type of `cache`. For empty entries we can just use `nil` (with an explicit assignment, like you already do, so the key would exists and we could look up for it). And we could avoid type conversions.

Reply:
Right, I was using a different empty struct type before but forgot to change this back.